### DISCUSSION TITLE
Cannot load file in some subdirectory

### DISCUSSION THREAD

**Discussion by nerocrux** [coddingtonbear/obsidian-csv-table#27] [Q&A] [answered]
The file can be shown if it is put under the root of the vault path.
However, if I create a subdirectory (i.e. "assets") and put the csv file into it, the plug in cannot load it properly.
Is it because of my OS's default language?

<img width="286" alt="スクリーンショット 2023-05-24 103410" src="https://github.com/coddingtonbear/obsidian-csv-table/assets/452535/66e3ee59-4518-4078-8d32-e5593cf21ee8">

**Comment by Z446C** [comment, marked as answer]
hello @nerocrux !
you can!
example, I have a demo.csv file in folder of obsidian project root.  ({root}/asset/demo.csv)
then, as follows
```csvtable
source: asset/demo.csv
```
